python/tests/test_collections.py

```diff
@@ -0,0 +1,68 @@
+from unittest.mock import Mock
+
+import pytest
+from numerous import collection
+from numerous._client import Client
+from numerous.generated.graphql.client import Client as GQLCleint
+from numerous.generated.graphql.fragments import CollectionNotFound, CollectionReference
+
+
+ORGANIZATION_ID = "test_org"
+COLLECTION_NAME = "test_collection"
+NESTED_COLLECTION_NAME = "nested_test_collection"
+COLLECTION_REFERENCE = CollectionReference(key="test_key", id="test_id")
+NESTED_COLLECTION_REFERENCE=CollectionReference(key="nested_test_key",
+                                                 id="nested_test_id")
+
+
+
+@pytest.fixture(autouse=True)
+def _set_env_vars(monkeypatch:pytest.MonkeyPatch)->None:
+    monkeypatch.setenv("NUMEROUS_API_URL", "url_value")
+    monkeypatch.setenv("NUMEROUS_API_ACCESS_TOKEN", "token")
+
+
+def test_collection_new_key()->None:
```

Comment:
You changed the test names, but they are still not indicating what is being tested here. What is the purpose of the test?

Furthermore tests need to be separated into sections of "arrange", "act" and "assert" where possible. Make it clear what are the preconditions, what is actual the thing being tested, and what are the assertions. Just make an empty line between these! This comment applies to all the tests.